### PR TITLE
feat:[TRI-1685]: added example values to open api yaml, fixed problem…

### PR DESCRIPTION
**Pull request by ds-psosnowski**
## Description
Added example values to open-api documentation and also did some clean up.
**Note:** Instead of using OpenApiExamples class I had to use direct examples in Schema annotation, since Open Api did not generate it correctly.  

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [ ] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [ ] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

File: irs-models/src/main/java/org/eclipse/tractusx/irs/component/assetadministrationshell/ProtocolInformation.java [hunk 1]
```diff
@@ -40,14 +42,17 @@ public class ProtocolInformation {
     /**
      * endpointAddress
      */
+    @Schema(implementation = String.class, example = "https://catena-x.net/vehicle/basedetails")
```
Comment (ds-jhartmann) [inline irs-models/src/main/java/org/eclipse/tractusx/irs/component/assetadministrationshell/ProtocolInformation.java:45]: I recommend to use an example value for href similar to https://eclipse-tractusx.github.io/docs-kits/kits/Traceability%20Kit/Software%20Development%20View/Specification%20Traceability%20Kit#submodel-descriptors

File: irs-models/src/main/java/org/eclipse/tractusx/irs/component/Relationship.java [hunk 2]
```diff
@@ -51,8 +51,10 @@ public class Relationship {
 
     private LinkedItem linkedItem;
 
+    @Schema(implementation = String.class, example = "supply_chain_impacted")
```
Comment (ds-jhartmann) [inline irs-models/src/main/java/org/eclipse/tractusx/irs/component/Relationship.java:54]: Example for a Relationship aspectType would be "SingleLevelBomAsBuilt".

**Review by ds-jhartmann — CHANGES_REQUESTED** [review]

File: irs-models/src/main/java/org/eclipse/tractusx/irs/component/assetadministrationshell/ProtocolInformation.java (hunk 1 above)
Comment (ds-jhartmann) [reply irs-models/src/main/java/org/eclipse/tractusx/irs/component/assetadministrationshell/ProtocolInformation.java]: Sorry for the confusion, I meant the example found on this site 😄 
"href": "https://edc.data.plane/{path}/submodel"

Comment (ds-psosnowski) [reply irs-models/src/main/java/org/eclipse/tractusx/irs/component/assetadministrationshell/ProtocolInformation.java]: oh sorry, I've just copied it 😄 fixing it right away

**Review by ds-jhartmann — APPROVED** [review]

**Pull request merged**